haiku-apps/xrs/xrs-1.9.1.recipe

```diff
@@ -0,0 +1,54 @@
+SUMMARY="A simple rhythm station for creating music loops"
+DESCRIPTION="XRS is a simple Rhythm Station. You can use it to create music compositions and loops of your preferred elettronic music genres."
```

Comment:
DESCRIPTION can only be 100 characters long, start a new line before that with ` \`